submissions/kt-std/oop-exercise/index.js

```diff
@@ -0,0 +1,115 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+  Complete the below for code reviewers' convenience:
+
+  Code repository: https://github.com/kt-std/kottans-frontend/tree/main/task_tiny_js_world
+  Web app: https://kt-std.github.io/a-tiny-JS-world/
+  */
+
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+
+/** Class representing an inhabitant. */
+class Inhabitant {
+  /**
+   * Create an inhabitant
+   * @param {string} species - The species of the inhabitant.
+   * @param {string} name - The name of the inhabitant.
+   * @param {string} gender - The gender of the inhabitant.
+   * @param {number} legs - The amount of legs of the inhabitant.
+   * @param {string} sayings - The text of inhabitant greetings.
+   * @param {[]} friends - The list of inhabitant friends.
+   */
+  constructor(species, name, gender, legs, sayings, friends) {
+    this.species = species;
+    this.name = name;
+    this.gender = gender;
+    this.legs = legs;
+    this.sayings = sayings;
+    this.friends = this.haveFriends(friends);
+  }
+
+  /**
+   * Check if an inhabitant has friends
+   * @param {[]} friends - The list of inhabitant friends
+   * @return {string} List of friends or sad quote
+   */
+  haveFriends(friends) {
+    return Array.isArray(friends) && friends.length ?
+      friends.join(', ') :
+      'Forever alone...';
+  }
```

Comment:
Storing friends as a string has a drawback:
- adding more friends during lifetime will be comlicated
~- what if we want to print not only friends' names but their other properties?~
Nevermind you use strings as friends rather than objects.

Just call this method in stringify instead of just referring `this.friends`

Reply:
Thank you so much for reviewing!  I've just fixed it.